src/pages/life-members/index.tsx

```diff
@@ -0,0 +1,141 @@
+import getLifeMembers from "@/api/modules/lifeMember/getLifeMembers";
+import getLifeMembersPageContent from "@/api/modules/lifeMember/getLifeMembersPageContent";
+import LineMember, { LineMemberProps } from "@/components/lineage/line_member";
+import Member from "@/components/member/member";
+import PageTemplate from "@/components/PageTemplate";
+import { GetServerSideProps } from "next";
+import Head from "next/head";
+import { v4 } from "uuid";
+
+export default function LifeMembers({
+  lifeMembers,
+  opening_paragraph
+}: {
+  lifeMembers: LifeMembers;
+  opening_paragraph: string
+}) {
+  const notFound =  <div>Life Members Not Found ...</div>;
```

Comment:
Maybe wrap in page template so that user can continue navigating with ease if data not found.

Reply:
We discussed... @shooboxx will create the charles I "data not found" component...aka much different from the page not found page